Makefile

```diff
@@ -34,9 +34,11 @@ Cargo.toml: Cargo.dev.toml
 
 dev-format: Cargo.toml
 	cargo fmt --all
+	zepter format features --fix
```

Comment:
maybe have them on a separate target otherwise we need to indicate on README to tell user to install `zepter` first

but a good amount of time people won't need to use it so I want to make it optional

Reply:
Okay I put it into another target. LMK if i overdid it 😅